### DISCUSSION TITLE
Welcome to ChurchApps Discussions!

### DISCUSSION THREAD

**Discussion by jzongker** [ChurchApps/ChurchAppsSupport#307] [General]
## Welcome to GitHub Discussions for Our Open Source Church Projects!

Hello everyone,

We are thrilled to announce that GitHub Discussions is now enabled for ChurchApps projects designed to support churches and their communities. This new space is dedicated to fostering collaboration, offering support, requesting new features, and sharing your unique setups and experiences.

### What You Can Do Here:

1. **Get and Offer Support**: Encountered an issue? Need guidance? Post your questions here, and let’s work together to find solutions. If you’ve overcome challenges, share your knowledge and help others.

2. **Request Features**: Have an idea that could improve our projects? Submit your ideas and participate in discussions about potential new functionalities.

3. **Share Setups**: Showcase how you’re using our projects in your church. Whether it’s a unique configuration, an innovative use case, or tips and tricks, your insights can inspire and assist others.

### How to Get Started:

1. **Explore the Categories**: Navigate through our discussion categories to find the right place for your post. We’ve organized them to make it easy to find and share relevant content.

2. **Join the Conversation**: Don’t be shy! Introduce yourself in our welcome thread, respond to ongoing discussions, or start a new topic that interests you.

3. **Stay Respectful and Helpful**: We are committed to maintaining a positive and supportive community. Please be respectful and constructive in your interactions.

### Why GitHub Discussions?

GitHub Discussions provides a central place for our community to engage more effectively. It integrates seamlessly with our project repositories, making it easier to track and participate in conversations that matter to our development and community support.

We look forward to seeing how this new platform enhances our collaboration and helps us serve churches better through our open source projects. Thank you for being part of this journey and for your continued support.

**Comment by I-NuKeS-I** [comment]
Good morning. So I have been test driving B1 church app and I the following issues, or maybe I didn't set it up right. But are the issues the following. Notifications in the mobile app never come in when I click notify volunteers, the app continues to be in English only and not in Spanish. It's for my Spanish speaking volunteers. The other issue is that the app logs you out every time.

I really want to move the whole church to use B1. Hopefully these issues will be resolved soon
